toc/TOC.md

```diff
@@ -115,3 +115,26 @@ approval, based on community feedback.
 
 The initial content of this page is from the work of the [Knative community](https://github.com/knative/community)
 under the terms of the [Creative Commons Attribution 4.0 License](https://creativecommons.org/licenses/by/4.0/).
+
+```yaml
+name: Technical Oversight Committee
+execution_leads:
+- name: Lee Porte
+  github: leeporte
+- name: David Stevenson
+  github: dsboulder
+- name: Eric Malm
+  github: emalm
+- name: Jan von Löwenstein
+  github: loewenstein
+- name: Stephan Merker
+  github: stephanme
+areas:
+- name: CloudFoundry Community
+  repositories:
+  - cloudfoundry/community
+config:
+  github_project_sync:
+    mapping:
+      cloudfoundry: 31
```

Comment:
For this to work you will need to create an org scoped project: https://github.com/orgs/cloudfoundry/projects?type=beta

The current one is a repo scoped one.

Reply:
I'm seeing it on the org level projects, Called CF Community

Reply:
I see, it seems to redirect, maybe they changed something. Since it redirects to the org scope, this code will probably work just fine: https://github.com/rkoster/github-multi-repo-project-card-sync/blob/main/github/client.go#L50-L62